src/test/scala/com/softwaremill/realworld/common/AuthorizationSpec.scala

```diff
@@ -74,6 +75,16 @@ object AuthorizationSpec extends ZIOSpecDefault:
       headers = Map(),
       expectedError = "Invalid value for: header Authorization (missing)"
     ),
+    ArticleAuthTestParameters(
```

Comment:
no positive test?

Reply:
Yes, like in https://github.com/softwaremill/realworld-tapir-zio/pull/69 I wasn't sure if I could test it. But finally it worked.